content/blog/2016/2016-05-04-the-state-of-JAM

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "The State of Jenkins Area Meetup (JAM)"
+tags:
+- meetup
+- JAM
+- event
+author: alyssat
+---
+
+Recently, the Jenkins project announced the release of link:https://jenkins.io/blog/2016/04/26/jenkins-20-is-here/[Jenkins 2.0], a first major release after 10 years and 655 weekly releases. This has been a major milestone for Jenkins and its growing community of developers, testers, designers and other users in the software delivery process. 
+
+With its rising popularity and wide adoption, the Jenkins community continues to grow and evolve into the millions. Jenkins community meetup activity has risen to an all time high since the first Jenkins meetup which was established on August 23 2010, in San Francisco. 
+
+Over the last six months the link:http://www.meetup.com/pro/jenkins/[Jenkins Area Meetup (JAM) Groups] has grown from 5 to 30 groups with coverage in Asia, North America, South America and Europe - collectively.  That’s an average growth of 4 new JAM groups per month.
+
+image:https://raw.githubusercontent.com/hinman55/jenkins.io/master/JAM-map.png[JAM map,500,float="right"]
```

Comment:
This isn't what @oleg-nenashev meant when he said add the images to the repository. If you add the repositories to your "patch-4" branch, they can (and should) then be referred to relatively since they would be packaged up in the generated site.

For example, [this blog post](https://jenkins.io/blog/2016/04/15/the-need-for-pipeline/) uses images which are referred to [here](https://github.com/jenkins-infra/jenkins.io/blame/master/content/blog/2016/2016-04-15-the-need-for-pipeline.adoc#L142). Those images are checked into the [`/content/images/post-images/need-for-pipeline/`](https://github.com/jenkins-infra/jenkins.io/tree/master/content/images/post-images/need-for-pipeline) directory.

In your fork of this repository, you should be adding the images you need to a new subdirectory of `post-images` and referring to them with a relative path similar to the post I linked above.